src/auth-socket-middleware.ts

```diff
@@ -19,17 +20,9 @@ export default function registerAuthSocketMiddleware(io: Server) {
       }
       logger.info(`Token decoded: ${JSON.stringify(decoded, null, 2)}`);
 
-      if (
-        decoded && 
-        typeof decoded !== 'string' && 
-        decoded.payload && 
-        decoded.payload.uuid
-      ) {
-        const uuid = decoded.payload.uuid;
-        logger.info(`user: ${uuid} is listening notifications`);
-        socket.join(uuid);
-      } else {
-        const email = typeof decoded === 'string' ? decoded : decoded!.email;
+      const jwtPayload = (decoded as Jwt)?.payload;
+      if (typeof jwtPayload === 'object' && jwtPayload.email) {
+        const email = jwtPayload.email;
```

Comment:
No, it is not necessary. I will leave the condition for using the email as it was.

Reply:
are we going to have troubles with this and this feature? @sg-gs https://inxt.atlassian.net/browse/PB-610

Reply:
Yes, we are going to have trouble. I would change whatever the server is joining with the email and do it properly rather than extending the issue here @CandelR @PixoDev